Documentation/network/servicemesh/ingress-to-gateway/ingress-to-gateway.rst

```diff
@@ -0,0 +1,109 @@
+including HTTP routing, TLS termination, traffic splitting/weighting, and header modification.
+
+Native support for these features means annotations are no longer needed to provide support for features that are commonly required
+for ingress traffic patterns. This means that Gateway API resources are more portable from one Gateway API implementation to another.
+
+When customization is required, Gateway API provides several flexible models, including specific extension points to enable diverse 
+traffic patterns. As extensions are added, the Gateway API team will keep looking for common denominators and will promote features 
+to the API conformance to avoid going back to some of the chaos seen with extending Ingress API resources.
+
+Finally, the Gateway API implementation was designed with role-based persona in mind. The Ingress model was based on a model where 
+it was assumed that developers managed and created Ingress and Services resources themselves.
+
+In many of the more complex deployments however, there are various personas involved:
+
+- Infrastructure Provider: responsible for the administration of the underlying infrastructure. For example, the cloud provider (AWS, Azure, GCP) when using 
+  their managed services or the infrastructure/network team when running Kubernetes on-premises.
+- Cluster Operator: responsible for the the administration of a cluster.
+- Application Developer: responsible for defining application configuration and service composition.
+
+By decomposing the Ingress API into several Gateway API objects, personas will be assigned the right access and privileges that their responsibilities require.
+
+For example, application developers in a specific team would be allowed to create Route objects in a specified namespace 
+but would not be able to modify the Gateway configuration or edit Route objects in namespaces other than theirs.
+
+Migration Methods
+#################
+
+There are currently two primary methods to migrate Ingress API resources to Gateway API:
+
+- *manual* - manually creating Gateway API resources based on existing Ingress API resources.
+- *automated*, creating rules using the `ingress2gateway tool <https://github.com/kubernetes-sigs/ingress2gateway>`_. 
+  This project reads Ingress resources from a Kubernetes cluster based on your current Kube Config. It will output YAML for equivalent Gateway API resources to stdout.
+  Note the ``ingress2gateway`` tool remains experimental at this stage.
+
+Ingress Annotations Migration
+#############################
+
+Most Ingress controllers make the use of annotations to provide support for specific features, such as HTTP request manipulation and routing. 
+As highlighted in the earlier section about its benefits, the Gateway API model avoids the use of implementation-specific annotations where possible to provide a portable configuration.
+
+Therefore, porting implementation-specific Ingress annotations to a Gateway API resource rarely applies. 
```

Comment:
Use clear, plain language

This doesn't make sense to me as written. Isn't the point of migration this exact scenario, to port from Ingress annotations to Gateway API resources? 
```suggestion
As a consequence, it's rare to port implementation-specific Ingress annotations to a Gateway API resource. 
```